sources.json

```diff
@@ -242,6 +242,78 @@
     "tags": ["odata", "promise"],
     "license": "mit"
   },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-error-collector",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["error","collector","exception","log","event"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-smart-mockserver",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["mock","mockserver","mockdata","faker","fake","smart","smarter"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-tour",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["tour","guide","intro","showcase"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-model-json-crud",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["model","JSON","CRUD"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-validator",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["validate","validator","validation"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-password",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
+    "tags": ["password","input","control"],
+    "license": "mit"
+  },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/ui5-deployer",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "tooling",
```

Comment:
@mauriciolauffer @marianfoo : I'm a bit unsure about the type for the ui5-deployer. It is a custom tooling extension - but doesn't include a task or a middleware. @uxkjaer - does this matter to your `add` generator? Generally spoken, this is a tooling extension but would naturally be a command extensions which is still missing. 

If ok with @uxkjaer for the `add` generator that this is being ignored, I am fine with it...

Reply:
Currently the add generator searches for tooling, tasks and middlewares, however it will try to figure out from the yaml file whether the tooling handles middleware or tasks. This could break the generator if you tried to add the deployer as isn't a task or middleware. Not much [code](https://github.com/ui5-community/generator-ui5-add/blob/main/src/App.ts#L445) needs to be updated though.

Happy for any pull requests to support this. And if you add the typedef to your index file, then it will generate the suggestion ;)

Reply:
I've never used `generator-ui5-add`, I don't know how it works.

Why listing it in `bestofui5` would break `generator-ui5-add`? Does it just pull and add everything listed on `bestofui5`?

Reply:
I would suggest we name this type `binary` and @marianfoo adds the type and a proper icon for it. 

@mauriciolauffer : the generator-ui5-add lists all `tooling`, `middleware` and `task` entries from `BestOfUI5`. As `tooling` is a container for `task` and `middleware` extensions this would lead to conflicts otherwise.

Reply:
Link to `generator-ui5-add`: https://github.com/ui5-community/generator-ui5-add/

Reply:
IMO, `binary` isn't a good name. I'd prefer to keep it as `tooling`, it seems clearer for me. It's most likely people  will search for ui5 tools in general using `tooling` as keyword, rather than `binary`.

Perhaps the fix should be on the `generator-ui5-add` to consider both keywords `tooling` + `task` and/or `middleware`. If it's just `tooling`, does nothing.

Reply:
Right now the name `tooling` manifests in the naming schema of aggregated tasks and middlewares . IMO, this would lead to misunderstandings what toolings are. The term `tooling` is the short name for tooling extensions. As the deployer is providing a custom binary and not integrated into the UI5 tooling (only via the configuration in the `ui5.yaml`) it is an individual solution and therefore a binary might be more fitting IMO. In future this may become a `command` extension - maybe using `command` would be the other option. WDYT?

Reply:
You can change it to `command`
Will do the changes to backend/frontend tomorrow

Reply:
Done   =)